### ISSUE TITLE
Winget failed to install

### ISSUE THREAD

**Issue by MrAbidPatel** [ChrisTitusTech/winutil#912]
Been running the utility flawlessly on my PCs at home and work in the last few months and managed to roll out winget to my PCs using the PS code (as well as directly using asheroto's script). However, since yesterday, I cannot get winget to install via the scripts. When I try to install anything using the utility, I get this error:

```
====Chris Titus Tech=====
=====Windows Toolbox=====
Checking if Chocolatey is Installed...
Chocolatey Already Installed
Checking if Winget is Installed...
Winget not found, installing it now.
===========================================
--      Winget failed to install        ---
===========================================
```

When I try asheroto's script directly, I get the following:

```
WARNING: Failed getting latest Microsoft.UI.Xaml version from API                                                       
https://api.nuget.org/v3-flatcontainer/Microsoft.UI.Xaml/index.json
WARNING: Using the hardcoded version 2.8.4

##################################
# Downloading Xaml nupkg file... #
##################################

Downloading: https://www.nuget.org/api/v2/package/Microsoft.UI.Xaml/2.8.4
Saving as  : C:\Users\Abid\AppData\Local\Temp\Microsoft.UI.Xaml.2.8.4.nupkg.zip

Expand into: C:\Users\Abid\AppData\Local\Temp\Microsoft.UI.Xaml.2.8.4

##########################################
# Downloading & installing x64 VCLibs... #
##########################################


##########################
# Installing x64 XAML... #
##########################

Installing Appx Packages In: C:\Users\Abid\AppData\Local\Temp\Microsoft.UI.Xaml.2.8.4\tools\AppX\x64\Release

Installing Appx Package: Microsoft.UI.Xaml.2.8.appx

#########################
# Downloading winget... #
#########################

Retrieving download URL for winget from GitHub...

Downloading: https://github.com/microsoft/winget-cli/releases/download/v1.5.1881/Microsoft.DesktopAppInstaller_8wekyb3d8bbwe.msixbundle
Saving as  : C:\Users\Abid\AppData\Local\Temp\winget.msixbundle


Downloading: https://github.com/microsoft/winget-cli/releases/download/v1.5.1881/c6bee58ade21451eb54dc52e559a314c_License1.xml
Saving as  : C:\Users\Abid\AppData\Local\Temp\license1.xml


########################
# Installing winget... #
########################

wingetPath       : C:\Users\Abid\AppData\Local\Temp\winget.msixbundle
wingetLicensePath: C:\Users\Abid\AppData\Local\Temp\license1.xml


#################################################################################################
# Checking and adding WindowsApps directory to PATH variable for current user if not present... #
#################################################################################################

C:\Users\Abid\AppData\Local\Microsoft\WindowsApps already present in PATH variable for current user, skipping.

##################
# Cleaning up... #
##################


##########################
# Installation complete! #
##########################


##############################################################
# Please restart your computer to complete the installation. #
##############################################################
```

Despite several reboots, I cannot get winget to install via the scripts. It works if I install it manually, but does anyone have any ideas why it won't install using the scripts? I have to use scripts to install winget as I want to deploy winget to my PCs remotely.

**Comment by KonTy** [comment]
@MrAbidPatel what version of Windows did you use? Also is it a Tiny11 flavor? It seems that some of the .net dependencies are missing from your system, somehow it got messed up on your system.

**Comment by MrAbidPatel** [comment]
> @MrAbidPatel what version of Windows did you use? Also is it a Tiny11 flavor? It seems that some of the .net dependencies are missing from your system, somehow it got messed up on your system.

It's Windows 10 Education (22H2), downloaded from the VLSC and then edited myself using NTLite. I don't remove any components for the production environment, mainly just settings tweaks and updates.

However, the last time I created the image was April 2023 (using Microsoft's April 2023 updated ISO) and the scripts were working fine on that image until this week.

The test from the code above was from a July 2023 updated ISO (SW_DVD9_Win_Pro_10_22H2.9_64BIT_Eng_Intl_Pro_Ent_EDU_N_MLF_X23-51693.ISO) that I created for testing. But I can't get winget to install on either build anymore when using the script.

I literally posted on the NTLite forums last week about how I managed to get winget integrated into the Windows image using asheroto's code from Chris' script and I was using the same method to deploy internally using PDQ Deploy and also using the install applications stage of MDT. Suddenly, it's stopped working.

**Comment by MrAbidPatel** [comment]
> Did you run it as admin ?

Yes, I did.

**Comment by ChrisTitusTech** [comment]
Fixed in commit #917 today

**Comment by MrAbidPatel** [comment]
> Fixed in commit #917 today

Thanks Chris 👍

**Comment by oziau** [comment]
WARNING: Winget was not found due to un-availablity reasons
===========================================
---      Winget is not installed        ---
===========================================

1. Windows 11 pro
2.  Powershell 7.4.4 as ADMIN
